app/controllers/admin/attachments/quarantined_attachments_controller.rb

```diff
@@ -78,6 +74,13 @@ def find_attachment
       rescue ActiveRecord::RecordNotFound
         render_404
       end
+
+      def breadcrumb_items
+        [{ href: admin_index_path, text: t("label_administration") },
+         { href: admin_settings_attachments_path, text: t("attributes.attachments") },
```

Comment:
I am not sure, whether we want the "Attachments" as an extra element. For the project attributes we did not add the intermediate step. I will double check with the product team